src/components/smart/AdresCrab.vue

```diff
@@ -311,7 +311,7 @@ import {
 } from '@govflanders/vl-ui-design-system-vue3';
 import type { IAdresCrabProps } from '@models/adres-crab';
 import type { IAdres, IGemeente, ILand, ILocatieAdres, IPostinfo, IStraat } from '@models/locatie';
-import { CrabService } from '@services/crab.api-service';
+import { CrabApiService } from '@services/crab-api.service';
```

Comment:
Is die naam 'Crab' eigenlijk nog wel goed? Want we hebben alles van crab er toch uitgefaseerd nu, toch?

Reply:
voor erna:
https://github.com/OnroerendErfgoed/vue_component_library/issues/33